portal-web/test/functional/com/liferay/portalweb/macros/FormRules.macro

```diff
@@ -296,6 +296,20 @@ definition {
 			locator1 = "FormRules#RULE_BUILDER_BROKEN");
 	}
 
+	macro viewBuilderRuleAction {
```

Comment:
@Diogomes, please rename to `viewRuleBuilderAction` to keep consistency with the name used in the UI and in the path `RULE_BUILDER_FIELD_GENERIC`